objcgen/objcgenerator.cs

```diff
@@ -367,6 +367,35 @@ protected override void Generate (Type t)
 				builder.WriteImplementation ();
 			}
 
+			if (equals.TryGetValue (t, out m)) {
+				var builder = new EqualsHelper (headers, implementation) {
+					ObjCSignature = $"isEqual:(id)other",
+					AssemblyName = aname,
+					MetadataToken = m.MetadataToken,
+					ObjCTypeName = managed_name,
+					ManagedTypeName = t.FullName,
+				};
+
+				builder.WriteHeaders ();
+				builder.WriteImplementation ();
+
+
+				GenerateGetGCHandle ();
+			}
+
+			if (hashes.TryGetValue (t, out m)) {
+				var builder = new HashHelper (headers, implementation) {
+					ObjCSignature = $"hash",
```

Comment:
no need for `$` here - even if, _I suspect_, the C# compiler won't call `Format` unless needed